tardis/plasma/properties/nlte.py

```diff
@@ -45,20 +45,18 @@ def set_initial_value(self, kwargs):
 class HeliumNLTE(ProcessingPlasmaProperty):
     outputs = ('helium_population',)
 
-    def calculate(self, level_boltzmann_factor, electron_densities,
+    @staticmethod
```

Comment:
Remnant from the original method I tried to get this to work before I thought of a different way. I can change it.